src/sema/statements.rs

```diff
@@ -1236,7 +1295,9 @@ fn destructure_values(
                 diagnostics,
                 Some(&Type::Bool),
             )?;
-
+            std::println!("Aqui!!");
+            std::println!("{:?}", cond);
```

Comment:
Please remove your debugging :smiley:

Reply:
As a curiosity, `aqui` means `here` in Portuguese :)